src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -130,6 +130,64 @@ void PreStepsList::add_mean_impl(const std::vector<float>& values) {
         "mean " + vector_to_string(values));
 }
 
+void PreStepsList::add_pad_impl(const std::vector<int>& pads_begin,
+                                const std::vector<int>& pads_end,
+                                const std::vector<float>& values,
+                                PaddingMode mode) {
+    std::string name;
+    name = "pad(begin " + vector_to_string(pads_begin) + 
```

Comment:
name = "pad(begin 

Here is open bracket but closing one is only when padding mode is constant. Should be added always?